assets/src/js/country_selector.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * The script basically add/remove the tabindex attribute within the country selector
+ * It helps to improve SEO
+ * @return {void}
+ */
+export const setupCountrySelector = () => {
+  const targetNode = document.querySelector('#country-selector');
+  const countriesList = document.querySelector('.countries-list');
+  const footerMenu = document.querySelector('.footer-menu');
+  const footerSocialMedia = document.querySelector('.footer-social-media');
+
+  if (!targetNode || !countriesList || !footerMenu || !footerSocialMedia) {
+    return;
+  }
+
+  const largeAndUpScreens = '(min-width: 992px)';
+  const countriesListTransitionDuration = (
+    parseFloat(getComputedStyle(document.querySelector('.countries-list')).transitionDuration) * 1000
+  ) || 500;
+
+  /**
+   * This funtion re-order the elements from the footer
+   * @param {*} evt
+   */
+  const switchOrders = (evt = null) => {
+    const event = evt ? evt : window.matchMedia(largeAndUpScreens);
+    if (event.matches) {
+      footerMenu.parentNode.insertBefore(footerMenu, footerMenu.parentNode.firstChild);
+    } else {
+      footerSocialMedia.parentNode.insertBefore(footerSocialMedia, footerSocialMedia.parentNode.firstChild);
+    }
+  };
+
+  // Create an observer instance linked to the callback function
+  const observer = new MutationObserver(
+    // Callback function to execute when mutations are observed
+    mutationList => {
+      // Set default visibility
+      for (const mutation of mutationList) {
+        if (mutation.attributeName === 'class') {
+          if (targetNode.classList.contains('open')) {
+            countriesList.style.visibility = 'visible';
+          } else {
+            setTimeout(() => {
+              countriesList.style.visibility = 'hidden';
+            }, countriesListTransitionDuration);
+          }
+        }
+      }
+    }
+  );
+
+  // Start observing the target node for configured mutations
+  observer.observe(targetNode, {attributes: true, childList: true, subtree: true});
```

Comment:
Do we need a mutation observer?
Is it not enough with an `onClick` listener?

Reply:
It could be potentially another option. In that case you might handle 2 different listeners. One for the open button, another for the close button and make sure to not propagate. If you think that that is a better approach I can work on that and re-implement the logic. What do you think?

Reply:
From the performance perspective, I think 2 `onClick` listeners are better than an observer continuously monitoring changes, but I am not sure. I also think that listeners are easier to read, understand, and maintain. Whatever you choose to do will be okay with me.

Reply:
Yep, I see your point! But being honest, I don't see a (potential) performance issue within this implementation. But of course we can follow up this implementation in the future during the `performance and page speed` project.